nemo_text_processing/text_normalization/taggers/cardinal.py

```diff
@@ -35,128 +35,14 @@ class CardinalFst(GraphFst):
     def __init__(self):
         super().__init__(name="cardinal", kind="classify")
 
-        delete_space = pynutil.delete(" ")
-        graph_zero = pynini.string_file(get_abs_path("data/numbers/zero.tsv"))
-        graph_digit = pynini.string_file(get_abs_path("data/numbers/digit.tsv"))
-        graph_ties = pynini.string_file(get_abs_path("data/numbers/ties.tsv"))
-        graph_teen = pynini.string_file(get_abs_path("data/numbers/teen.tsv"))
-
-        delete_extra_spaces = (
-            pynini.closure(pynutil.delete(" "))
-            + pynini.closure(pynini.closure(NEMO_ALPHA, 1) + delete_extra_space)
-            + pynini.closure(NEMO_ALPHA, 1)
-            + pynini.closure(pynutil.delete(" "))
-        )
-
-        graph_hundred = pynutil.delete("hundred")
-
-        graph_hundred_component = pynini.union(
-            graph_digit + delete_space + graph_hundred + delete_space, pynutil.insert("0")
-        )
-        graph_hundred_component += pynini.union(
-            graph_teen | pynutil.insert("00"),
-            (graph_ties + delete_space | pynutil.insert("0")) + (graph_digit | pynutil.insert("0")),
-        )
-
-        #  string -> all 3 digit numbers apart from 000
-        graph_hundred_component_at_least_one_none_zero_digit = graph_hundred_component @ (
-            pynini.closure(NEMO_DIGIT) + (NEMO_DIGIT - "0") + pynini.closure(NEMO_DIGIT)
-        )
-
-        # all 3 digit numbers apart from 0 -> string
+        graph = pynini.Far(get_abs_path("data/numbers/cardinal_number_name.far")).get_fst()
```

Comment:
is this docstring still valid: `-23 -> cardinal { negative: "true" integer: "twenty three" } }`

Reply:
yep